samples/system-test/automlVision.test.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright 2018, Google, LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const path = require(`path`);
+const storage = require(`@google-cloud/storage`)();
+const test = require(`ava`);
+const tools = require(`@google-cloud/nodejs-repo-tools`);
+const uuid = require(`uuid`);
+
+const bucketName = `nodejs-docs-samples-test-${uuid.v4()}`;
+const cmdDataset = `node ./automl/automlVisionDataset.js`;
+const cmdModel = `node ./automl/automlVisionModel.js`;
+const cmdPredict = `node ./automl/automlVisionPredict.js`;
+
+const flowerTrainData = `gs://`;
+const cwd = path.join(__dirname, `..`);
+
+const testDataSetName = `testDataSet`;
+const dummyDataSet = `dummyDataSet`;
+const testModelName = "dummyModel";
+const sampleImage = `./automl/resources/testImage.jpg`;
+const sampleImage2 = `./automl/resources/testImage2.jpg`;
+
+// skipped because it's been taking too long to delete datasets
+test.skip(`It should create a create, list, and delete a dataset`, async t => {
```

Comment:
Delete `a create`?

Also, ideally each of these (`create`, `read`, and `delete`) would be in a separate test. You can use `test.serial()` to mark tests as having to run in-order. (The one downside is that it can slow down your test runs.)